tests/golden.rs

```diff
@@ -74,6 +74,14 @@ fn encode_ic11() {
     encoder_test("32x32.png", IconType::RGBA32_16x16_2x, "ic11.icns");
 }
 
+#[test]
+fn decode_ic13() {
+    decoder_test("ic13.icns", IconType::RGBA32_128x128_2x, "256x256.png");
```

Comment:
Out of curiosity, where did the ICNS file for this new test come from?

Reply:
I just looped through every icon on my computer till I found one that contained jpeg2000. I have no idea what the icon actually is from.

Reply:
It appears it isn't actually jp2k and I just assumed it was, rather strange, i'll test it again with an actual jp2k

Reply:
@mdsteele I have added an actual jp2k icns (ic08) and a test for it, and it appears not to work.